### ISSUE TITLE
Add Suite layer to Refiner

### ISSUE THREAD

**Issue by mikesaintsg** [japa/core#47]
Version 5.0.17-0

I assumed that a suite would be another layer in the Refiner, but I was a bit surprised to find that it wasn't.

I was able to hack in userland a way to add suites to the runner only when it's listed, but it feels like it should already be included. Especially when we want to open the runner up to plugins.

Let me know your thoughts, thanks!

**Comment by thetutlage** [comment]
Yeah. Suites are outside the refiner on purpose. Because, this is something the high level runner should be handling.

Let's understand the flow from outside in.

- You create a tests runner called `run-it`.
- The developer writing the tests can provide the suite name using some CLI flag. `run-it --suites=unit`.
- Now, within the codebase of `run-it`, you will have to make sure to not even import the files for suites other than `unit`.

So basically, refiner doesn't have much to filter in the case of suites. The logic of importing the right files (as per the suites filter) must be handled by the high level runner itself.

**Comment by mikesaintsg** [comment]
Thanks! That makes sense, the suites should be filtered out before it even reaches the runner. It wouldn't make sense to let it reach so far down into the Refiner logic.
Much appreciated!
